sdk/key_vault/src/key.rs

```diff
@@ -296,6 +576,105 @@ impl<'a, T: TokenCredential> KeyClient<'a, T> {
         result.algorithm = algorithm;
         Ok(result)
     }
+
+    /// Decrypt a single block of encrypted data.
+    /// The DECRYPT operation decrypts a well-formed block of ciphertext using the target encryption key and specified algorithm.
+    /// This operation is the reverse of the ENCRYPT operation; only a single block of data may be decrypted, the size of this block is dependent on the target key and the algorithm to be used.
+    /// The DECRYPT operation applies to asymmetric and symmetric keys stored in Vault or HSM since it uses the private portion of the key. This operation requires the keys/decrypt permission.
+    pub async fn decrypt(
+        &mut self,
+        key_name: &str,
+        key_version: Option<&str>,
+        decrypt_parameters: DecryptParameters,
+    ) -> Result<DecryptResult, Error> {
+        // POST {vaultBaseUrl}/keys/{key-name}/{key-version}/decrypt?api-version=7.2
+
+        let mut uri = self.vault_url.clone();
+        let path = if let Some(ver) = key_version {
+            format!("keys/{}/{}/decrypt", key_name, ver)
+        } else {
+            format!("keys/{}/decrypt", key_name)
+        };
```

Comment:
Nit: you actually don't need to do this. The service backend already treats version as optional, so even consecutive "//" won't matter.